testutils/Makefile

```diff
@@ -6,7 +6,7 @@ GOPROXY ?=
 export GOPROXY
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.28
+ENVTEST_K8S_VERSION = 1.31
```

Comment:
1.32.1? or are we not using the openshift builds for a reason here 🤔

Reply:
We should, originally I didn't bump the whole repo to 1.32 for CAPI compat reasons.
Now that I did, this should be updated too.

Pushed a change to do this.